build-gluster-org/jobs/32-bit-build-smoke.yml

```diff
@@ -1,7 +1,7 @@
 - job:
     name: gh_32-bit-build-smoke
     node: fedora30
-    description: smoke job for 32 bit build
+    description: smoke job for 32 bit build - https://github.com/gluster/build-jobs/blob/master/build-gluster-org/scripts/32-bit-build-smoke.sh
```

Comment:
should we change default 'branch' of build-jobs repo before doing it? so we can save one more PR later?

Reply:
Agreed. I am switching the repo to the new default branch.